driver-core/src/main/com/mongodb/ConnectionString.java

```diff
@@ -1157,6 +1164,17 @@ public Boolean getRetryWritesValue() {
         return retryWrites;
     }
 
+    /**
+     * <p>Gets whether reads should be retried if they fail due to a network error</p>
+     *
+     * @return the retryWrites value, or true if unset
+     * @since 3.11
+     * @mongodb.server.release 3.6
+     */
+    public Boolean getRetryReads() {
```

Comment:
Let the MongoClient determine the default if it's unset, and just return `retryReads` value directly here.

Reply:
Done.